.bazelversion

```diff
@@ -1 +1 @@
-6.0.0-pre.20220706.4
+5.0.0
```

Comment:
This looks like merge error

Reply:
yeah I'm not sure how that snuck in; done.